tests/analysis/adaptation/test_adaptation_option_collection.py

```diff
@@ -67,15 +67,25 @@ def test_calculate_options_unit_cost_returns_dict(
         assert isinstance(_result, dict)
         assert all(_option in _result for _option in _collection.adaptation_options)
 
-    def test_calculate_options_benefit_returns_series(self):
+    def test_calculate_options_benefit_returns_gdf(self):
         @dataclass
         class MockOption:
             # Mock to avoid the need to run the impact analysis.
             id: str
             impact: float
 
+            @property
+            def benefit_col(self) -> str:
+                return f"{self.id}_benefit"
+
+            @property
+            def impact_col(self) -> str:
+                return f"{self.id}_impact"
```

Comment:
Same, to be a proper mock it should at least inherit from the same protocol / base class.

Reply:
Adjusted/simplified